pyomo/contrib/fme/tests/test_fourier_motzkin_elimination.py

```diff
@@ -61,7 +61,7 @@ def test_no_vars_specified(self):
     def check_projected_constraints(self, m):
         constraints = m._pyomo_contrib_fme_transformation.projected_constraints
         # x - 0.01y <= 1
-        cons = constraints[4]
+        cons = constraints[1]
```

Comment:
Why did the resulting constraints change order?

Reply:
They changed because I changed the order the remaining constraints appear in after one iteration of FME, when I stopped using `pop` on the list of constraints. Specifically, `waiting_list` comes first now, and then the new constraints. It was the opposite before. It should still be deterministic, so I think it's okay.

Reply:
Great.  That's fine as long as we know why the order changed (and we didn't let some nondeterminism sneak in).